scrapeAndClean/BeautifulSoup/functions.py

```diff
@@ -59,8 +59,10 @@ def clean_df(df):
     df_header = ['Incident', 'Date Occurred', 'Location', 'Details']
     df.columns = df_header
     # Remove the redundant part in Details column
-    start_pattern = 'If you have any information related to this incident'
-    df['Details'] = [x[:x.find(start_pattern)] for x in df['Details']]
+    # Commenting out the following two lines of code because it has some
```

Comment:
Records with problems were: 7/15/2017 and 10/20/2017
There could be more records with issues as well